build.gradle

```diff
@@ -18,6 +20,7 @@ apply plugin: 'eclipse'
 apply plugin: 'maven-publish'
 apply plugin: 'java'
 apply plugin: 'checkstyle'
+apply plugin: 'com.adarshr.test-logger'
 

```

Comment:
This plugin makes it easier to look at the results of unit tests
![Uploading image.png…]()